htdocs/js/apps/Stats/stats.js

```diff
@@ -0,0 +1,124 @@
+(() => {
+	// Hover action to show exact value of each bar.
+	document.querySelectorAll('.bar_graph_bar').forEach((el) => {
+		el.addEventListener('mousemove', (evt) => {
+			const tooltip = document.getElementById('bar_tooltip');
+			tooltip.innerHTML = evt.target.dataset.tooltip;
+			tooltip.style.display = 'block';
+			tooltip.style.left = evt.pageX + 10 + 'px';
+			tooltip.style.top = evt.pageY + 10 + 'px';
+		});
+		el.addEventListener('mouseout', () => {
+			document.getElementById('bar_tooltip').style.display = 'none';
+		});
+	});
```

Comment:
The final step for making the Bootstrap tooltips work.  Change this to:
```suggestion
	// Tooltip to show exact value of each bar.
	document.querySelectorAll('.bar_graph_bar').forEach((el) => new bootstrap.Tooltip(el, { trigger: 'hover' }));
```